src/common/assets/generalData/fetchedData.js

```diff
@@ -1,16 +1,18 @@
-// import movie_id from "../../../features/movies/"
-export const API_KEY = process.env.REACT_APP_API_KEY
+const API_KEY = process.env.REACT_APP_API_KEY
```

Comment:
IMO this const has to be exported otherwise there will be a problem with compiling. I'm getting an error: "ERROR in ./src/features/movies/MoviePage/moviePageSaga.js 11:53-60
export 'API_KEY' (imported as 'API_KEY') was not found in '../../../common/assets/generalData/fetchedData' (possible exports: URL, URLimage, URLmovie, URLpeople, URLpopularMovies, URLpopularPeople)"